code/game/objects/items/stacks/medical.dm

```diff
@@ -11,96 +11,40 @@
 	resistance_flags = FLAMMABLE
 	max_integrity = 40
 	novariants = FALSE
-	var/heal_brute = 0
-	var/heal_burn = 0
-	var/stop_bleeding = 0
+	item_flags = NOBLUDGEON
 	var/self_delay = 50
 
 /obj/item/stack/medical/attack(mob/living/M, mob/user)
-
-	if(M.stat == DEAD && !stop_bleeding)
-		var/t_him = "it"
-		if(M.gender == MALE)
-			t_him = "him"
-		else if(M.gender == FEMALE)
-			t_him = "her"
-		to_chat(user, "<span class='danger'>\The [M] is dead, you cannot help [t_him]!</span>")
+	. = ..()
+	if(!M.can_inject(user, 1))

```

Comment:
what does the 1 mean

Reply:
It generates an error message on humans if aimed at bodypart is somehow blocked. This is always true on all other carbons/living and this check returns false on silicons.

I removed the message afterwards and replaced the 1 with a TRUE. (even though it's somewhat stupid that the error message gets generated in human... guess that's something for another day)